src/resources/i18n/translations/en.json

```diff
@@ -810,5 +810,10 @@
   "Your queue is empty": "Your queue is empty",
   "Your membership will not auto-renew": "Your membership will not auto-renew",
   "You are not being charged during your free trial": "You are not charged during or after the free trial.",
-  "You will not be charged for signing up for your free trial": "You are not charged for signing up or using your free trial. We hope you subscribe (manually) afterwards."
+  "You will not be charged for signing up for your free trial": "You are not charged for signing up or using your free trial. We hope you subscribe (manually) afterwards.",
+  "Android Auto Podcast Title": "Podcast",
+  "Android Auto Queue Title": "Queue",
+  "Android Auto History Title": "History",
+  "Android Auto Permission Title": "Permission Required",
```

Comment:
We might need to explicitly say something like "Android Auto Permission Required"